<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Actogram</class>
 <widget class="QWidget" name="Actogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCustomPlot" name="actoplot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="gb_acto_range">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Actogram configuration options</string>
        </property>
        <property name="title">
         <string>Configuration</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Start Day</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="startDay">
             <property name="toolTip">
              <string>First day in actogram</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="dateTime">
              <datetime>
               <hour>16</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>2</month>
               <day>2</day>
              </datetime>
             </property>
             <property name="maximumTime">
              <time>
               <hour>20</hour>
               <minute>59</minute>
               <second>59</second>
              </time>
             </property>
             <property name="minimumTime">
              <time>
               <hour>19</hour>
               <minute>0</minute>
               <second>0</second>
              </time>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="currentSectionIndex">
              <number>2</number>
             </property>
             <property name="timeSpec">
              <enum>Qt::UTC</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QScrollBar" name="sb_startday">
             <property name="toolTip">
              <string>Adjust start day</string>
             </property>
             <property name="pageStep">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Days</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="days">
             <property name="toolTip">
              <string>Number of days in actogram</string>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Plot Double</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="cb_double">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>PercentLimit</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="sb_actorange">
             <property name="toolTip">
              <string>Range of Y-axis (in activity percentage)</string>
             </property>
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>UTC Offset</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QSpinBox" name="sb_offset">
             <property name="toolTip">
              <string>Offset from UTC (hours)</string>
             </property>
             <property name="minimum">
              <number>-12</number>
             </property>
             <property name="maximum">
              <number>12</number>
             </property>
             <property name="value">
              <number>-5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Actogram Title</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="le_actogram">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Optional graph title</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Daylight</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="longitude">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
           <property name="value">
            <double>-79.218744000000001</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="rb_no_light">
           <property name="text">
            <string>None</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_lighting</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="latitude">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>-180.000000000000000</double>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
           <property name="value">
            <double>42.110599999999998</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Longitude</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="rb_natural_light">
           <property name="text">
            <string>Sun Elevation</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_lighting</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="rb_lab_light">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Load light Model from file</string>
           </property>
           <property name="text">
            <string>Light Model</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bg_lighting</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pb_load_lightfile">
           <property name="toolTip">
            <string>Load Light Model from file</string>
           </property>
           <property name="text">
            <string>Load </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_acto_export">
        <property name="toolTip">
         <string>Export functions</string>
        </property>
        <property name="title">
         <string>Export</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pb_acto_pdf">
           <property name="toolTip">
            <string>Export actogram to PDF</string>
           </property>
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_acto_png">
           <property name="toolTip">
            <string>Export actograph to PNG</string>
           </property>
           <property name="text">
            <string>PNG</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_lighting"/>
 </buttongroups>
</ui>
